server/src/main/java/org/candlepin/dto/api/v1/EnvironmentTranslator.java

```diff
@@ -64,35 +62,48 @@ public EnvironmentDTO populate(Environment source, EnvironmentDTO dest) {
     public EnvironmentDTO populate(ModelTranslator translator, Environment source,
         EnvironmentDTO dest) {
 
-        dest = super.populate(translator, source, dest);
+        if (source == null) {
+            throw new IllegalArgumentException("source is null");
+        }
+
+        if (dest == null) {
+            throw new IllegalArgumentException("destination is null");
+        }
 
-        dest.setId(source.getId());
-        dest.setName(source.getName());
-        dest.setDescription(source.getDescription());
+        Date created = source.getCreated();
+        dest.created(created != null ? created.toInstant().atOffset(ZoneOffset.UTC) : null);
+
+        Date updated = source.getUpdated();
+        dest.updated(updated != null ? updated.toInstant().atOffset(ZoneOffset.UTC) : null);
```

Comment:
One more minor change, that we could apply in general from now on: Use org.candlepin.util.Util.toDateTime() to as a utility converter from Date to OffsetDateTime

Reply:
Fixed.